learn/advanced/filtering.mdx

```diff
@@ -92,6 +92,8 @@ Meilisearch supports the following filter operators:
 - [Inequality `!=`](#inequality)
 - [Comparison `> < <= >=`](#comparison)
 - [`EXISTS`](#exists)
+- [`IS EMPTY`](#is-empty)
+- [`IS NULL`](#is-null)
 - [`IN`](#in)
 - [`NOT`](#not)
 - [`AND`](#and)
```

Comment:
I think we can remove this list, WDYT? It was a good idea when we didn't have the "on this page sidebar" allowing users to jump between operators without scrolling